game/src/sandbox/dashboards/mod.rs

```diff
@@ -18,9 +18,7 @@ mod trip_table;
 // Oh the dashboards melted, but we still had the radio
```

Comment:
You were so close to my Modest Mouse reference, but no comment about it? :P

Reply:
All post-2000 MM references are lost on me, but I'll check it out as penance. :neckbeard: